pandas/tests/io/formats/test_format.py

```diff
@@ -2740,6 +2740,14 @@ def test_format_percentiles():
         fmt.format_percentiles([0.1, 0.5, 'a'])
 
 
```

Comment:
can you add an additional test that uses .describe, like you did in the OP: https://github.com/pandas-dev/pandas/issues/26660#issue-452509070, add in pandas/tests/frame/test_analytics

Reply:
I've added the test you mentioned. Please comment if it is not sufficient.